lib/bamboo/email.ex

```diff
@@ -113,6 +115,16 @@ defmodule Bamboo.Email do
     struct!(%__MODULE__{}, attrs)
   end
 
+  for function_name <- @address_functions do
+    @doc """
+    Sets the #{function_name} on the email
```

Comment:
Maybe add an example?

```elixir
    email |> #{function_name}("chase@gmail.com")
```

It would be good to run `mix docs` to make sure it looks ok though